src/server.rs

```diff
@@ -331,6 +338,20 @@ async fn check_webui(config: Arc<RwLock<Configuration>>) -> Result<(), Rejection
     } else {
         Err(reject::not_found())
     }
+}
+
+fn sse_event_stream(
+    events: Events,
+) -> impl Stream<Item = Result<impl ServerSentEvent + Send + 'static, warp::Error>> + Send + 'static
+{
+    let my_id = NEXT_EVENT_ID.fetch_add(1, Ordering::Relaxed);
+
+    let (tx, rx) = mpsc::unbounded_channel();
+
+    tx.send(Message { data: String::from("lol") }).unwrap();

```

Comment:
@CephalonRho I don't understand these two lines if I remove it it does not works :/